com.unity.shadergraph/Editor/Data/Nodes/CodeFunctionNode.cs

```diff
@@ -277,65 +277,65 @@ public sealed override void UpdateNodeAfterDeserialization()
             {
                 AddSlot(slot);
             }
-            RemoveSlotsNameNotMatching(slots.Select(x => x.id));
+            RemoveSlotsNameNotMatching(slots.Select(x => x.id), true);
         }
 
-        private static MaterialSlot CreateBoundSlot(Binding attributeBinding, int slotId, string displayName, string shaderOutputName, ShaderStageCapability shaderStageCapability, bool hidden)
```

Comment:
Is this change related?  Or just setting up nice defaults?

Reply:
change related because it wasnt getting passed in before (which would default to false when making the slots) so I just wanted to keep it as consistent as possible